src/components/SelectInput.jsx

```diff
@@ -0,0 +1,28 @@
+import { IoChevronDownOutline } from 'react-icons/io5';
+
+const SelectInput = ({ id, label }) => {
+	return (
+		<div className="relative">
+			<label htmlFor={id} className="text-xs font-medium">
+				{label}
+			</label>
+			<select
+				id={id}
+				className="cursor-pointer appearance-none bg-white shadow hover:shadow-md text-gray-900 text-sm rounded-lg focus:tcl-blue focus:tcl-blue block p-2.5 h-[42px] w-[300px]"
+			>
+				<option value="" disabled selected>
```

Comment:
This gives this error for me:
`Warning: Use the 'defaultValue' or 'value' props on <select> instead of setting 'selected' on <option>`. Something that I found that works without the error is having 
`defaultValue={value}` in the select on lines 9-13. Then in the AddItem component having
`<SelectInput label="Timeframe" id="timeframe" value={itemValue.daysUntilNextPurchase}/>`

Reply:
Thanks Aloe! Yeah I did notice that and I forgot about it when it came time for my PR. I solved it like so all in the same component:
```
  <select
	defaultValue="disabled"
  >
  <option value="disabled" disabled>
       Select an option
  </option>
```